thermal_image_topic_name: /thermal_camera/image_raw
camera_frame_id: thermal_camera
camera_name: thermal_camera
camera_info_url: file://${ROS_HOME}/camera_info/${NAME}.yaml
color_mode: color
interpolate_dead_pixels: true
calibrate_dead_pixels: true
mean_compensation: false
calibrate_mean_compensation: false
denoise: true
show_debug_images: true
